content/learn/05.communication/01.wire/wire.md

```diff
@@ -1,30 +1,92 @@
 
-- Arduino IDE ([online](https://create.arduino.cc/) or [offline](https://www.arduino.cc/en/main/software)).
-- 2x Arduino Boards
+Both Qwiic and STEMMA QT bundle together wires for power, ground, as well as the SDA and SCL wires for I2C, making it a complete kit, one cable that bundles everything together.
+
+### What's the difference between the two?
+Both Qwiic and STEMMA QT use I2C, and even when inspecting modules using the two standards up close, it may be difficult to tell what makes them unique from each other. But there is a difference! And it has some implications on how and for what you may use them.
+
+Qwiic has level shifting and voltage regulation on the controller (but not on the peripherals). What this means is that Qwiic is 3.3 V logic **only**. This makes it easier to use, as for the end user, there is one less thing that can go wrong when designing and assembling your circuit.
+
+STEMMA QT, on the other hand, doesn't have this. This lets you use both 3.3 V and 5 V logic for modules. This also means that there is one more thing you may need to consider when creating your circuit, but it also grants some more flexibility in power and logic requirements.
+
+The pin order for STEMMA QT is designed to match the pin order for Qwiic, enabling cross-compatibility between the two standards
+
+## Wire Library
+The Wire library is what Arduino uses to communicate with I2C devices. It is included in all board packages, so you don't need to install it manually in order to use it.
+
+To see the full API for the Wire library, visit its [documentation page](https://www.arduino.cc/reference/en/language/functions/communication/wire/).
+
+- `begin()` - Initialise the I2C bus
+- `end()` - Close the I2C bus
+- `requestFrom()`- Request bytes from a peripheral device
+- `beginTransmission()` - Begins queueing up a transmission 
+- `endTransmission()` - Transmit the bytes that have been queued and end the transmission 
+- `write()` - Writes data from peripheral to controller or vice versa
+- `available()` - returns the number of bytes available for retrieval
+- `read()` - Reads a byte that was transmitted from a peripheral to a controller.
+- `setClock()` - Modify the clock frequency
+- `onReceive()` - Register a function to be called when a peripheral receives a transmission
+- `onRequest()` - Register a function to be called hwen a controller requests data
+- `setWireTimeout()` - Sets the timeout for transmissions in controller mode
+- `clearWireTimeoutFlag()` - Clears the timeout flag
+- `getWireTimeoutFlag()` - Checks whether a timeout has occurred since the last time the flag was cleared.
+
+## Examples
```

Comment:
So there's no examples in this section, just circuits. I think we can move this to a new section further up, and call it "Arduino I2C Pins", and just provide these circuits.